cedar-policy-core/src/evaluator/err.rs

```diff
@@ -30,9 +29,9 @@ pub enum EvaluationError {
 
     /// Tried to get this attribute, but the specified entity didn't
     /// have that attribute
-    #[error("{} does not have the required attribute: {}", &.entity, &.attr)]
+    #[error("`{}` does not have the attribute: {}", &.entity, &.attr)]
     EntityAttrDoesNotExist {
-        /// Entity which didn't have the attribute
+        /// Entity that didn't have the attribute
```

Comment:
Nit: didn't -> doesn't

Reply:
All the surrounding comments are past tense, so I'd prefer to leave as-is.